features/login_github_redirect.feature

```diff
@@ -0,0 +1,151 @@
+        | user3 |                 | test@user.com |
+
+
+  And I am not logged in
+
+Scenario: not logged in so should see login with Github from users page
+  #Story ID: #152298585
+  Given I am on the users page
+  Then I should see "Log in with GitHub"
+ 
+Scenario: not logged in so should see login with Github from orgs page 
+  #Story ID: #152298585
+  Given I am on the orgs page
+  Then I should see "Log in with GitHub"
+  
+Scenario: not logged in so should see login with Github from create page 
+  #Story ID: #152298585
+  Given I am on the create page
+  Then I should be on the login page
+  # Then I should see "Log in with GitHub"
+
+Scenario: login with Github from orgs page and redirect to orgs
+  #Story ID: #152298585
+  Given I am not logged in
+  And I am on the apps page
+  And I follow "Orgs"
+  And I follow "Log in with GitHub"
+  Then I should be on the orgs page
+
+Scenario: login with Github from users page and redirect to users
+  #Story ID: #152298585
+  Given I am not logged in
+  And I am on the apps page
+  And I follow "Users"
+  #And I am visiting the provider "users"
+  And I follow "Log in with GitHub"
+  Then I should be on the users page
+
+Scenario: login with Github from create page and redirect to create
+  #Story ID: #152298585
+  Given I am not logged in
+  And I am on the apps page
+  Given I follow "Create"
+  And I follow "Log in with GitHub"
+  Then I should be on the create page
+  
+Scenario: login with Github from create page, logout, login back and redirect to create
+  #Story ID: #152298585
+  Given I am not logged in
+  And I am on the apps page
+  And I follow "Create"
+  And I follow "Log in with GitHub"
+  And I am not logged in
+  And I follow "Log in with GitHub"
+  Then I should be on the create page
+  
+Scenario: login with Github from create page, after visiting orgs and users
+  #Story ID: #152298585
+  Given I am not logged in
+  And I am on the apps page
+  And I follow "Create"
+  And I follow "Orgs"
+  And I follow "Users"
+  And I follow "Create"
+  And I follow "Log in with GitHub"
+  Then I should be on the create page
+  
+Scenario: Login to github to create a new app
+  #Story ID: #152298585
+  Given I am not logged in
+  And I am on the apps page
+  And I follow "New App"
+  And I am on the login page
+  And I follow "Log in with GitHub"
+  And I am on the new_app page
+  And I press "Save"
+  And I should see "2 errors prohibited this app from being saved:"
+  And I should see "App Name can't be blank"
+  And I should see "App Description can't be blank"
+  When I fill in "App Name" with "Fake app"
+  When I fill in "App Description" with "Fake app description "
+  When I fill in "Deployment url" with "Fake app deployment url"
+  When I fill in "Repository url" with "Fake app repository "
+  And I press "Save"
+  Then I should be on the apps page
+  And I should see "App was successfully created."
+
+Scenario: Login to github to edit an existing app successfully
```

Comment:
Same here. Check only if the user is indeed redirected to edit_app page.

Reply:
Thanks for the suggestion! I kept the login part of the test in the login feature and moved the apps_controller test part to a new feature for apps_controller.